lib/core/nuxt.js

```diff
@@ -129,7 +129,13 @@ export default class Nuxt {
             return reject(err)
           }
 
-          const listenURL = chalk.underline.blue(`http://${host}:${port}`)
+          const connection = server.address()
```

Comment:
How about  const { address: host, port }= server.address()

Reply:
`port` was already defined and I wasn't sure what the nuxt stance on nesting variables was. I'll update it.